internal/providers/terraform/parser.go

```diff
@@ -648,6 +631,28 @@ func (p *Parser) parseConfReferences(resData map[string]*schema.ResourceData, co
 	return found
 }
 
+func (p *Parser) parseTags(data map[string]*schema.ResourceData, providerConf gjson.Result, conf gjson.Result) {
+	for _, resourceData := range data {
+		resConf := getConfJSON(conf, resourceData.Address)
+		defaultTags := parseDefaultTags(providerConf, resConf)
```

Comment:
These two lines are related to AWS only. Can we move them under `case "aws"`?

Reply:
good point